### PR TITLE
add event funding page

### PR DESCRIPTION
**Pull request by domenkozar**
Still needs styling.

### CODE REVIEW

**Review by refroni — COMMENTED** [review]
Looks good :)

File: community/event-funding.tt [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+<h2>Can we apply for recurring events?</h2>
+<p>We prefer you apply for each instance to be fair to other events happening but you may apply as a larger event that is spread across the year. The bar for larger events/funding is higher.</p>
+
+<h2>What’s the approval process like at the board?</h2>
+<ol>
+  <li>Collection of all responses between the submission times</li>
+  <li>Board will review the requests and prioritize</li>
+  <li>
+    Examples of events we’re willing to sponsor (in that order)
```
Comment (fricklerhandwerk) [inline community/event-funding.tt:60]:
```suggestion
    Examples of events we’re willing to sponsor, by priority:
```

**Comment by garbas** [comment]
@domenkozar would you mind pushing the branch to the repo. that way we will all get a preview link :)

File: community/event-funding.tt [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    <ol>
+      <li>Events that are focused on advocating Nix to bring new community members</li>
+      <li>Events that are focused on building in Nix</li>
```
[the remark this answers is not in the document]
Comment (domenkozar) [reply community/event-funding.tt:63]: "building using the Nix ecosystem"?

**Comment by domenkozar** [comment]
https://github.com/NixOS/nixos-homepage/pull/990

**Pull request closed without merging**